src/classes/EP_Task_UTIL.cls

```diff
@@ -39,6 +39,9 @@
 */
 public class EP_Task_UTIL {
 
+    /** @description Additional Field Mappings found using UTIL_SimpleFieldMapper functionality. */
+    private Set<String> additionalFieldMappings;
```

Comment:
Instead of using the `getAdditionalFieldMappings();` and calling it separately from code, the var can be called instead and utilize lazy load.